src/utils/functions/getGoogleOauthURL.ts

```diff
@@ -1,7 +1,7 @@
 function getGoogleOauthURL(): string {
   const rootUrl = 'https://accounts.google.com/o/oauth2/v2/auth';
   const options = {
-    redirect_uri: 'http://localhost:5000/auth/google/redirect',
+    redirect_uri: 'http://localhost:5001/auth/google/redirect',
```

Comment:
why it here?

Reply:
Port 5000 already in use — MacOS Monterey issue